.gitignore

```diff
@@ -28,3 +28,49 @@ third-party
 
 # Build
 /*build*
+
```

Comment:
I'd suggest removing the below. If you cmake in a build directory (e.g., mkdir build; cd build; cmake .. && make -j) it should not generate these files.

Reply:
Good tip, thanks!